.github/workflows/fetcher.yml

```diff
@@ -1,13 +1,16 @@
 name: Fetcher
 
 on:
+  pull_request:
   push:
     branches: [ main ]
   schedule:
     - cron: '0 * * * *'
   workflow_dispatch:
 
 jobs:
+  gemfilesfetcher: 
+    uses: ietf-ribose/relaton-gemfiles-fetcher/.github/workflows/gemfiles-fetcher.yml@master
```

Comment:
@stefanomunarini Small suggestion, can you rename the `master` branch as `main` so it's in line with IETF inclusive language guidelines? Thanks.

Reply:
@kesara done! Sorry this PR was supposed to be in ietf-ribose repository and not in ietf-tools just yet. I'll close this for now for our internal review. Will open a new PR from ietf-ribose repository soon. Thanks

Reply:
No worries. 👍